numpy/typing/tests/data/pass/ufuncs.py

```diff
@@ -4,7 +4,7 @@
 np.sin([1, 2, 3])
 np.sin(1, out=np.empty(1))
 np.matmul(np.ones((2, 2, 2)), np.ones((2, 2, 2)), axes=[(0, 1), (0, 1), (0, 1)])
-np.sin(1, signature="D")
+np.sin(1, signature="D->D")
```

Comment:
@BvB93 I assme that this doesn't need any further changes related to typing, since typing probably doesn't make a difference between the one dtype vs. all dtypes `signature` versions?

Reply:
Naah, it's fine. `signature` is currently typed as taking either a string or tuple of strings, while this PR only changes which specific strings are allowed, correct?
https://github.com/numpy/numpy/blob/623bc1fae1d47df24e7f1e29321d0c0ba2771ce0/numpy/__init__.pyi#L2930